### PR TITLE
Update views related to policies

### PR DESCRIPTION
**Pull request by gayyappan**
Remove reorder and drop policies views
Add timescaledb_information.jobs view
Update job_stats and continuous_aggregates views.

This PR has 2 commits.

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -2923,67 +2927,111 @@ dex | orderby_asc | orderby_nullsfirst
...
+|`max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
+|`max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
+|`retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure |
+|`proc_schema` | (NAME) Schema name of the function or procedure executed by the job |
+|`proc_name` | (NAME) Name of the function or procedure executed by the job |
+|`owner` | (NAME) Owner of the job |
+|`scheduled` | (BOOLEAN) | Is the job scheduled to run automatically? |
+|`config` | (JSONB) | Configuration passed to the function specified by `proc_name` at execution time |
+|`next_start` | (TIMESTAMP WITH TIME ZONE) | Next start time for the job, if it is scheduled to run automatically |
+|`hypertable_schema` | (NAME) Schema name of the hypertable. NULL, if this is a user defined action.|
```
Comment (svenklemm) [inline api.md:2950]: also NULL for telemetry

**Review by svenklemm — APPROVED** [review]

File: api.md (hunk 1 above)
Comment (gayyappan) [reply api.md:2950]: Not sure if we want to call out Telemetry specifically. 
```
|`hypertable_schema` | (NAME) Schema name of the hypertable. NULL, if **this is Telemetry or a user defined action **.|
```

File: api.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 >:TOPLIST:
 > ### Command List (A-Z)
+> - [add_continuous_aggregate_policy](#add_continuous_aggregate_policy)
 > - [add_compression_policy](#add_compression_policy)
```
Comment (mfreed) [inline api.md:6]: Nit, alphabetize:
```suggestion
> - [add_compression_policy](#add_compression_policy)
> - [add_continuous_aggregate_policy](#add_continuous_aggregate_policy)
```

File: api.md [hunk 3]
```diff
@@ -42,6 +43,7 @@
 > - [locf](#locf)
 > - [move_chunk](#move_chunk)
 > - [refresh materialized view (continuous aggregate)](#continuous_aggregate-refresh_view)
+> - [remove_continuous_aggregate_policy](#remove_continuous_aggregate_policy)
 > - [remove_compression_policy](#remove_compression_policy)
```
Comment (mfreed) [inline api.md:47]: Nit, alphabetize:
```suggestion
> - [remove_compression_policy](#remove_compression_policy)
> - [remove_continuous_aggregate_policy](#remove_continuous_aggregate_policy)
```

**Review by mfreed — APPROVED** [review]

File: api.md (hunk 2 above)
Comment (gayyappan) [reply api.md:6]: fixed.

File: api.md (hunk 3 above)
Comment (gayyappan) [reply api.md:47]: fixed

**Pull request merged**